inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -230,6 +231,7 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
     pass_config->disable<ngraph::pass::LogSoftmaxDecomposition>();
     pass_config->disable<ngraph::pass::ConvertInterpolateToInterpOrResampleMatcher>();
     pass_config->disable<ngraph::pass::WeightsDequantizeToFakeQuantize>();
+    pass_config->disable<ngraph::pass::SimplifyCTCGreedyDecoderSeqLen>();
```

Comment:
Why is it called for the CPU plugin? It seems that CPU supports this operation doesn't it?

Reply:
We call transformation in common part for all plugins. For CPU we disable it.

Reply:
I see. Thanks.

Reply:
No, `pass_config->disable` disable this transformation for CPU.